repo/catalog.py

```diff
@@ -38,9 +38,13 @@ def write_catalog(config, feeds):
 		cat_root.setAttributeNS(XMLNS_NAMESPACE, 'xmlns:' + name, ns)
 		custom_tags[ns] = tags
 
-	for feed in feeds:
-		feed_root = feed.doc.documentElement
+	feed_roots = [feed.doc.documentElement for feed in feeds]
 
+	def get_name(feed_root):
+		return feed_root.getElementsByTagName('name')[0].firstChild.wholeText
+
+	for feed_root in sorted(feed_roots, key=get_name):
+		print(get_name(feed_root))
```

Comment:
Whoops, missed that debugging left-over. Removed it and updated the pull request.